src/screens/Inventory/components/PurchaseRequestCard.tsx

```diff
@@ -41,7 +38,8 @@ const styles = (theme: Theme) =>
 
 // TODO: sort by creation date/status
 function PurchaseRequestCard(props: PurchaseRequestCardProps) {
-  const { classes, product, amountPurchased, createdAt, requesterId, status } = props;
+  const { classes, inventoryId, amountPurchased, createdAt, requesterId, status } = props;
+  const product = useSelector((state: RootState) => selectProductById(state, selectCurrentSiteInventoryById(state, inventoryId)?.productId || "")) || EMPTY_PRODUCT;
```

Comment:
Can we separate this out to 2 lines? This line is 165 characters. Maybe have getting the currentSiteInventoryId the line before?